### ISSUE TITLE
Save filees won't load correctly

### ISSUE THREAD

**Issue by Pigable08** [Picorims/wav2bar#73]
**Describe the bug**
When you load a save file, the text below the "Import Save" button says it has imported, but nothing changed.

**To Reproduce**
1. Open the app
2. Click "Import Save"
3. Import any .w2bzip
4. Nothing happens

**Expected behavior**
I expected the save file to load my save and all my elements/objects.

**Screenshots**
![image](https://github.com/user-attachments/assets/f71cfe33-cc7b-433f-bd84-d88959b11223)
Note how the save says it has imported

**Desktop (please complete the following information):**
 - OS: Windows 11, (tried on Windows 10 as well)
 - Version 0.3.3
 - Iris XE Integrated Graphics

**Comment by Picorims** [comment]
Mmh that's annoying if that is the case.

Do you see an error in the logs? Did you try with a non empty save just to be sure? Are you sure you saved your changes?

Is it an installed version or a portable version?

**Comment by Picorims** [comment]
The save is a zip file so you can change the extension and check the JSON data inside by yourself. But if the JSON file is empty then something went wrong at saving. If you do this on the problematic save, do you find something in it?

**Comment by Pigable08** [comment]
No, I checked while it was loaded. However, on my saves now, loading works fine. It might just be that one. One reason could be the fact I created the save  on a school computer. These has limitations and we don't have admin. Could that cause it?

**Comment by Picorims** [comment]
If it tried to write a file and the computer denied it, that's perfectly possible that it didn't save. But I am deeply annoyed that it didn't warn you...

**Comment by Pigable08** [comment]
Thanks. Saves work with new files though so it could be perms. Hopefully the conclusion has been reached. Any more info you need just shout!

**Comment by susantpatel** [comment]
I am also facing same issue
Windows 11

Below is my logs

[2025-02-12T13:42:54.537] [INFO] main - Running Wav2Bar v0.3.3-beta
[2025-02-12T13:42:54.538] [WARN] main - Can't write in app's root folder. Writing in app data folder provided by the OS.
[2025-02-12T13:42:54.538] [INFO] main - Working directory: C:\Wav2Bar
[2025-02-12T13:42:54.538] [INFO] main - creating main renderer...
[2025-02-12T13:42:54.570] [INFO] main - main renderer created.
[2025-02-12T13:42:54.827] [DEBUG] main_renderer - Loading modules done.
[2025-02-12T13:42:54.827] [INFO] main_renderer - loaded default save.
[2025-02-12T13:42:54.827] [DEBUG] main_renderer - is export window: false
[2025-02-12T13:42:54.828] [INFO] main_renderer - screen set to 1280x720
[2025-02-12T13:42:54.828] [INFO] main_renderer - animation stopped.
[2025-02-12T13:42:54.829] [INFO] main_renderer - FPS set to 60
[2025-02-12T13:42:54.829] [INFO] main_renderer - screen set to 1280x720
[2025-02-12T13:42:54.829] [INFO] main_renderer - screen set to 1280x720
[2025-02-12T13:42:54.829] [INFO] main_renderer - initializing settings...
[2025-02-12T13:42:54.829] [INFO] main_renderer - user settings found.
[2025-02-12T13:42:54.829] [DEBUG] main - reading C:\Wav2Bar/user/settings/user_settings.json...
[2025-02-12T13:42:54.830] [DEBUG] main - reading C:\Wav2Bar/user/settings/user_settings.json done.
[2025-02-12T13:42:54.831] [INFO] main_renderer - saving settings...
[2025-02-12T13:42:54.831] [DEBUG] main - writing json file to C:\Wav2Bar/user/settings/user_settings.json.
[2025-02-12T13:42:54.831] [INFO] main_renderer - loading settings...
[2025-02-12T13:42:54.831] [INFO] main_renderer - loaded settings!
[2025-02-12T13:42:54.831] [INFO] main_renderer - settings loaded:
[2025-02-12T13:42:54.831] [INFO] main_renderer - {"save_version":1,"software_version_used":"0.3.3-beta","ffmpeg":{"ffmpeg_path":"C:\\Wav2Bar\\ffmpeg-7.1-essentials_build\\bin\\ffmpeg.exe","ffprobe_path":"C:\\Wav2Bar\\ffmpeg-7.1-essentials_build\\bin\\ffprobe.exe"}}
[2025-02-12T13:42:54.831] [DEBUG] main - reading ./assets/help/help.json...
[2025-02-12T13:42:54.832] [DEBUG] main - wrote json file to C:\Wav2Bar/user/settings/user_settings.json.
[2025-02-12T13:42:54.832] [DEBUG] main - reading ./assets/help/help.json done.
[2025-02-12T13:42:54.832] [INFO] main_renderer - settings saved!
[2025-02-12T13:42:57.347] [DEBUG] main - reading directory C:\Users\susan
[2025-02-12T13:42:57.352] [DEBUG] main - reading directory C:\Users\susan done
[2025-02-12T13:42:59.399] [DEBUG] main - reading directory 
[2025-02-12T13:42:59.401] [ERROR] main_renderer -  do not exists: Error: Error invoking remote method 'read-dir': Error: read-dir: impossible to read this directory: Error: ENOENT: no such file or directory, scandir ''
[2025-02-12T13:42:59.702] [DEBUG] main - reading directory S
[2025-02-12T13:42:59.703] [ERROR] main_renderer - S do not exists: Error: Error invoking remote method 'read-dir': Error: read-dir: impossible to read this directory: Error: ENOENT: no such file or directory, scandir 'C:\Program Files (x86)\Wav2Bar\app-0.3.3-beta\resources\app\S'
[2025-02-12T13:43:00.038] [DEBUG] main - reading directory S:
[2025-02-12T13:43:00.041] [DEBUG] main - reading directory S: done
[2025-02-12T13:43:02.176] [DEBUG] main - reading directory S:\M&M Local
[2025-02-12T13:43:02.179] [DEBUG] main - reading directory S:\M&M Local done
[2025-02-12T13:43:03.828] [INFO] main_renderer - Backing up currently opened save...
[2025-02-12T13:43:03.828] [INFO] main_renderer - generating save file...
[2025-02-12T13:43:03.828] [DEBUG] main - writing json file to C:\Wav2Bar/temp/current_save/data.json.
[2025-02-12T13:43:03.828] [INFO] main - creating save file at C:\Wav2Bar/temp/before_new_save_open.w2bzip
[2025-02-12T13:43:03.829] [INFO] main_renderer - save file generated!
[2025-02-12T13:43:03.829] [INFO] main_renderer - Closing audio context if any...
[2025-02-12T13:43:03.829] [INFO] main_renderer - Audio context closed.
[2025-02-12T13:43:03.829] [INFO] main_renderer - Loading the save...
[2025-02-12T13:43:03.829] [INFO] main - loading save file at S:\M&M Local\demowav.w2bzip
[2025-02-12T13:43:03.831] [DEBUG] main - wrote json file to C:\Wav2Bar/temp/current_save/data.json.
[2025-02-12T13:43:03.831] [ERROR] main - failed at creating the save file: Error: ENOENT: no such file or directory, stat 'C:\Wav2Bar\temp\current_save\data.json'
[2025-02-12T13:43:03.845] [INFO] main_renderer - reading the save...
[2025-02-12T13:43:03.845] [DEBUG] main - reading C:\Wav2Bar/temp/current_save/data.json...
[2025-02-12T13:43:03.851] [DEBUG] main - reading C:\Wav2Bar/temp/current_save/data.json done.
[2025-02-12T13:43:03.851] [INFO] main_renderer - applying save...
[2025-02-12T13:43:03.851] [INFO] main_renderer - animation stopped.
[2025-02-12T13:43:03.851] [INFO] main_renderer - FPS set to 60
[2025-02-12T13:43:03.852] [INFO] main_renderer - screen set to 1280x720
[2025-02-12T13:43:03.852] [INFO] main_renderer - Save loaded!
[2025-02-12T13:43:14.899] [WARN] main - opening C:\Wav2Bar\logs.
